src/core/Microsoft.PowerToys.Settings.UI.Lib/ViewModels/GeneralViewModel.cs

```diff
@@ -242,11 +252,13 @@ public bool IsDarkThemeRadioButtonChecked
                     {
                         UpdateUIThemeCallBack(GeneralSettingsConfig.Theme);
                     }
+#pragma warning disable CA1031 // Do not catch general exception types
```

Comment:
This isn't really a good pattern.  We should do our best to understand what scenarios we expect exceptions to be thrown, and if its unknown, at the very least we should log the exception.

Reply:
@ryanbodrug-microsoft I'm beginning to want to cut this out into its own PR, because on further investigation I think there are different classes of exceptions that should be handled differently and that warrants its own discussion. Meanwhile most of the other changes here can be addressed easily if needed, so I'll remove this and then we can re-review this PR

Reply:
Sounds good to me.